components/brave_rewards/browser/rewards_service.h

```diff
@@ -312,6 +312,8 @@ class RewardsService : public KeyedService {
 
   virtual void GetExternalWallet(GetExternalWalletCallback callback) = 0;
 
+  virtual std::string GetExternalWalletType() const = 0;
```

Comment:
I'd prefer that we not make this public if possible, as it will likely need to change when we add support for Gemini. Does it work to use the "type" field of `mojom::ExternalWallet` instead?